experimental/split_grpc/proxy/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "split_grpc_proxy"
+version = "0.1.0"
+authors = ["Conrad Grobler <grobler@google.com>"]
+edition = "2018"
+
+[dependencies]
+env_logger = "0.7"
+futures = "0.3"
+log = "0.4"
+tokio = { version = "0.2", features = ["full"] }
```

Comment:
I personally don't believe in semver, so I have been using "*" everywhere, up to you if you think it makes sense to specify anything specific here.

Reply:
I have changed it to *. I have been following the examples for tonic and tokio, and all the samples were using specific versions. I assumed it was because of API incompatibility between different versions, but I guess if specific versions are locked down in the cargo.lock file it should not matter.